client/index.html

```diff
@@ -120,7 +120,7 @@ <h1 class="title">Connect</h1>
 								<div class="col-sm-3"></div>
 								<div class="col-sm-9">
 									<label class="tls">
-										<input type="checkbox" name="tls" <%= defaults.tls ? 'checked="checked"' : '' %>>
+										<input type="checkbox" name="tls" <%= defaults.tls ? 'checked="checked"' : '' %> <%= typeof(lockNetwork) !== "undefined" && lockNetwork ? 'disabled' : ''%>>
```

Comment:
I thought about that at first, but actually there is no need to escape here, unless I am being mistaken at that time of the day: `<%= typeof(lockNetwork) !== "undefined" && lockNetwork ? "disabled" : "" %>` seems to be fine to me.